toqito/matrix_props/is_pseudohermitian.py

```diff
@@ -0,0 +1,34 @@
+def is_pseudohermitian(mat: np.ndarray, signature_tup: tuple):
+    """
+    Checks if a matrix is pseudo-hermitian with respect to a given signature.
+
+    Args:
+        mat (np.ndarray): The matrix to check.
```

Comment:
If you type-hint in the function argument listing, this information in the doc-string is redundant and can be removed.